changelog/unreleased/issue-3090

```diff
@@ -0,0 +1,6 @@
+Bugfix: --use-fs-snapshot failed on windows/386
```

Comment:
Bugfix: --use-fs-snapshot now works on windows/386

Reply:
Fixed in https://github.com/restic/restic/pull/3090/commits/88208c3db27da33ab9141f10abf384b7a9045221